src/main/java/ShortSet.java

```diff
@@ -1,3 +1,5 @@
+import java.util.*;
+
 public class ShortSet {
```

Comment:
Good job! The static analysis tool Error Prone reported that the below error [1] used to be here, but you removed it. Error Prone also found similar issues in https://github.com/chbrown13/RecommenderTest/blob/37a92d8c9cf1f132a59dc2dea77b99baa3ebf25e/src/main/java/ShortSet.java#L3 and https://github.com/chbrown13/RecommenderTest/blob/37a92d8c9cf1f132a59dc2dea77b99baa3ebf25e/RecommenderTest/src/main/java/LongSet.java#L3.Check out http://errorprone.info for more information.


[1] ```RecommenderTest/src/main/java/ShortList.java:3: error: cannot find symbol
    List<Short> s = new ArrayList<>();
    ^
  symbol:   class List
  location: class ShortList```